cpp/velox/udf/Udaf.h

```diff
@@ -23,7 +23,7 @@ struct UdafEntry {
   const char* name;
   const char* dataType;
 
-  size_t numArgs;
+  int numArgs;
```

Comment:
Will this change matter? I see velox uses `uint32_t` in some places.

Reply:
It's a C interface, not C++. We should not use c++ types in it.

Reply:
see, not notice this~